sql_generators/glean_usage/event_error_monitoring.py

```diff
@@ -45,31 +46,39 @@ def generate_across_apps(
             and s.bq_table == "events_v1"
         ]
 
-        default_events_table = ConfigLoader.get(
+        default_event_table = ConfigLoader.get(
             "generate",
             "glean_usage",
             "events_monitoring",
             "default_event_table",
             fallback="events_v1",
         )
-        events_table_overwrites = ConfigLoader.get(
-            "generate", "glean_usage", "events_monitoring", "event_table", fallback={}
-        )
```

Comment:
This has been unused since https://github.com/mozilla/bigquery-etl/pull/5576/files